.lintrc

```diff
@@ -1,6 +1,7 @@
 # See https://github.com/markstory/lint-review
 [files]
 ignore = */migrations/*
```

Comment:
@emord should we remove this line? seems like a nice way to get @dimagimon to yell at us when we put migrations in the wrong place.